addons/modules/CfgVehicles.hpp

```diff
@@ -300,6 +300,59 @@ class CfgVehicles {
         displayName = CSTRING(ModulePatrolArea);
         function = QFUNC(modulePatrolArea);
     };
+    class GVAR(moduleToggleThrowTrajectory): GVAR(moduleBase) {
+        category = QGVAR(Projectiles);
+        displayName = CSTRING(moduleToggleThrowTrajectory);
+        function = QFUNC(moduleToggleThrowTrajectory);
+    };
+    class GVAR(moduleThrowFrag): GVAR(moduleBase) {
+        curatorCanAttach = 1;
+        category = QGVAR(Projectiles);
+		displayName = CSTRING(moduleThrowFrag);
+        function = QFUNC(moduleThrowFrag);
+    };
+    class GVAR(moduleThrowMini): GVAR(moduleBase) {
+        curatorCanAttach = 1;
+        category = QGVAR(Projectiles);
+		displayName = CSTRING(moduleThrowMini);
+        function = QFUNC(moduleThrowMini);
+    };
+    class GVAR(moduleThrowSmoke): GVAR(moduleBase) {
+        curatorCanAttach = 1;
+        category = QGVAR(Projectiles);
+		displayName = CSTRING(moduleThrowSmoke);
+        function = QFUNC(moduleThrowSmoke);
+    };
+    class GVAR(moduleThrowStone): GVAR(moduleBase) {
+        curatorCanAttach = 1;
+        category = QGVAR(Projectiles);
+		displayName = CSTRING(moduleThrowStone);
+        function = QFUNC(moduleThrowStone);
+    };
+    class GVAR(moduleThrowCan): GVAR(moduleBase) {
```

Comment:
Personally I don't think these should be in ZEN, but rather ALiVE. They should at least be hidden if ALiVE is not loaded.

Reply:
I could remove those until I figure out how to do that.